File: Java/UberBlack.java
import java.util.Map;
import java.util.ArrayList;

class UberBlack extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, 
    Map<String, Map<String,Integer>> typeCarAccepted, 
    ArrayList<String> seatsMaterial){
        super(license, driver); /*Super hara referencia a la clase padre */
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seetsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seetsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seetsMaterial = seetsMaterial